modules/acquia_cms_site_studio/composer.json

```diff
@@ -53,7 +50,8 @@
         "enable-patching": true,
         "patches": {
             "acquia/cohesion": {
-                "3340269 - Site Studio: Allow optional packages to import": "https://www.drupal.org/files/issues/2023-03-16/3340269-site-studio-allow-optional-packages-6.patch"
+                "3340269 - Site Studio: Allow optional packages to import": "https://www.drupal.org/files/issues/2023-03-16/3340269-site-studio-allow-optional-packages-6.patch",
+                "2571235 - Roles should depend on objects that are building the granted permissions": "https://gist.githubusercontent.com/vishalkhode1/e1ac95e3cbac97a06cd52430e3018b57/raw/af1f374c5263fc67f12b1f36b22144b57bc8714a/site-studio-dynamic-permissions.patch"
```

Comment:
@chandan-singh7929 We shouldn't add gist permissions in acquia_cms module composer.json. Please add in root composer.json.

Reply:
Done